docs/docs/getting-started.server.mdx

```diff
@@ -506,7 +506,47 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 through webpack loader syntax in imports.
 
 Install the webpack loader [`@mdx-js/loader`][mdx-loader].
-There is no need to configure it.
+
+To enable direct MDX imports w/o `!@mdx-js/loader!` prefix, the loader can
+further be added to webpack’s config, using a
+[`react-scripts`](http://github.com/facebook/create-react-app/tree/main/packages/react-scripts)
```

Comment:
```suggestion
`react-scripts`
```

I don’t think this has to be linked, as it does not add something useful to users that are trying to solve this problem, to open it in a tab?